doc/conf.py

```diff
@@ -291,6 +294,12 @@ def _check_dependencies():
 html_theme_options = {
     "logo_link": "index",
     "collapse_navigation": True if CIRCLECI else False,
+    # "external_links": [
```

Comment:
Did you want to keep this comment?

Reply:
It should go as I think it is extra-useless as this is read by `navbar_center` template that got replaced a bit further down.